File: sample/scala/app/controllers/Application.scala
package controllers

import play.api._
import play.api.mvc._
import play.api.libs.iteratee.Enumerator
import com.degiske.play2.chameleon.Chameleon

object Application extends Controller {

  def index(theme : String) = Action {
    val param = Array[Object]("Your theme is " + theme)
    SimpleResult(
      header = ResponseHeader(200, Map(CONTENT_TYPE -> "text/html")),
      body = Enumerator(Chameleon.render("views.html."+ theme + ".index", param).toString().getBytes())
    )
  }

}

File: sample/java/app/controllers/Application.java
package controllers;

import play.*;
import play.mvc.*;
import com.degiske.play2.chameleon.Chameleon;

import java.util.Map;
import java.util.Set;

public class Application extends Controller {

    public static Result index() {

        String theme = request().getQueryString("theme");

        if(null == theme)
        {
            theme = "theme1";
        }

        return ok(Chameleon.render("views.html."+ theme + ".index", new Object[]{"Your theme is " + theme}));
    }

}
